README.md

```diff
@@ -96,6 +96,7 @@ $ node example.js | pino
 * `logger`: `pino-http` can reuse a pino instance if passed with the `logger` property
 * `genReqId`: you can pass a function which gets used to generate a request id. The first argument is the request itself. As fallback `pino-http` is just using an integer. This default might not be the desired behavior if you're running multiple instances of the app
 * `useLevel`: the logger level `pino-http` is using to log out the response. default: `info`
+* `customLogLevel`: you can pass a function which gets used to determine the logger level. The first argument is the response. The second argument is the error object if it passed. If you pass this option don't pass `useLevel` option.
```

Comment:
Hmm, @mcollina do you think this is better named something like `levelHook` or `levelInterceptor`?

Either way, the text should be modified to:

> set to a function `(res, err) => { /* returns level name string */ }`. This function will be invoked to determine the level at which the log should be issued. This option is mutually exclusive with the `useLevel` option. The first argument is the HTTP response. The second argument is an error object if an error has occurred in the request.